### ISSUE TITLE
ANSI sequence "ESC [ 8;<lines>;<cols>t" fails to resize window in Terminal V1.4.3243.0, works in CMD.EXE (old console window)

### ISSUE THREAD

**Issue by pjfarleyiii** [microsoft/terminal#8673]
```
# Environment

Microsoft Windows [Version 10.0.19041.685]
Windows Terminal version: 1.4.3243.0

Any other software? No
```

# Steps to reproduce

In a non-maximized Terminal window, type (or execute a CMD file containing)
echo E[8;52;160t
where "E" is an "escape" character (X'1B')

# Expected behavior

Terminal window size should resize to 52 lines by 160 columns

# Actual behavior

No change in Terminal window size.
Old Windows Console (CMD.EXE) resizes to 52 columns by 160 columns.

Documentation: from https://invisible-island.net/xterm/ctlseqs/ctlseqs.html
CSI Ps ; Ps ; Ps t
  Ps = 8 ;  height ;  width ⇒  Resize the text area to given
          height and width in characters.  Omitted parameters reuse the
          current height or width.  Zero parameters use the display's
          height or width.

**Comment by DHowett** [comment]
Thanks! This is a /dupe of #5094

**Comment by Jibun-no-Kage** [comment]
Any progress on this issue?

**Comment by steviator** [comment]
Apparently Windows Terminal is abandoned now. In four years nobody from the mothership has even bothered to respond to this report.

Can anyone point me in the direction of a fork that is still actively developed, or should I revert to using conhost which still seems to be seeing improvements, and supports such advanced functionality?

**Comment by zadjii-msft** [comment]
@steviator buddy a PR was merged for #5094 over in #17721 like, a month ago. Terminal is still very actively maintained. I'd follow #5094 for updates, rather than a random duplicate thread.
